modular_skyrat/modules/Syndie_edits/code/syndie_edits.dm

```diff
@@ -11,3 +11,47 @@
 	armor = list(MELEE = 50, BULLET = 40, LASER = 50, ENERGY = 50, BOMB = 25, BIO = 0, FIRE = 100, ACID = 90, WOUND = 10)
 	dog_fashion = null
 	resistance_flags = FIRE_PROOF
+
+//DS2 items
+
+/obj/item/clothing/head/helmet/swat/ds
+	name = "interdyne SWAT helmet"
+	desc = "A robust and spaceworthy helmet with a small cross on it with an 'IP' written across the earpad."
```

Comment:
"spaceworthy" implies pressure resistance, dont say that or people will whine when their head explodes

Reply:
Original swat helmet actually is spaceproof. So it's basically just an armored EVA helmet.